pcsx2/gui/GlobalCommands.cpp

```diff
@@ -247,10 +248,11 @@ namespace Implementations
 			return;
 		g_Conf->GSWindow.Zoom = zoom;
 		
-		if ( zoom == 0 ) 
-            Console.WriteLn(L"(GSwindow) Zoom: 0 (auto, no black bars)");
-		else 
-            Console.WriteLn(L"(GSwindow) Zoom: %f", zoom);
+		if ( zoom == 0 ) {
+			OSDlog( Color_StrongBlue, true, "(GSwindow) Zoom: 0 (auto, no black bars)");
+		} else {
+			OSDlog( Color_StrongBlue, true, "(GSwindow) Zoom: %f", zoom);
+		}
```

Comment:
Not sure why you added braces here when they're not necessary? doesn't also make the code more readable in my opinion.